resources/views/backend/posts/show.blade.php

```diff
@@ -11,11 +11,16 @@
     </section>
     <section class="content">
       <div class="row">
+        @foreach ($posts as $post)
         <div class="col-md-3">
           <!-- Profile Image -->
           <div class="box box-primary">
             <div class="box-body box-profile">
-              <img class="img-thumbnail" src="{{ asset('images/books/no-image.png')}}" alt="User profile picture">
+                @if ($post->image == '')
+                  <img class="img-thumbnail" src="{{ asset('images/books/lrg.jpg')}}" alt="User profile picture">
+                @else
+                 <img class="img-thumbnail" src="{{ asset('images/books/'.$post->image)}}" alt="User profile picture">
+                @endif
```

Comment:
dùng Attribute thay cho if else nha a

Reply:
attribute la sao nghia. co the giai thich ko??